### PR TITLE
Write Simple Auton and add some Documentation

### LINKED ISSUE #7: Write SimpleAuton
Requires #6 

Write (and implement) a command (or more likely a command group) that is constructed with a time (in seconds) and a speed. 

The command should:
1. Drive forward at given speed
2. Stop after given time

### PR DESCRIPTION
**Pull request by aidan-mundy**
Closes #7 

Added some documentation for previously undocumented classes
Wrote and enabled the Simple Auton with placeholders for some of the constants

### CODE REVIEW

**Comment by kchenturtles** [comment]
Went over the Simple Auton Command new code
Read changed documentation to make sure it made sense

**Review by kchenturtles — DISMISSED** [review]
Went over the Simple Auton Command new code
Read changed documentation to make sure it made sense

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -6,6 +6,7 @@
 
 import edu.wpi.first.wpilibj.GenericHID;
 import edu.wpi.first.wpilibj.XboxController;
```
Comment (chit-sudeep) [inline src/main/java/frc/robot/RobotContainer.java:8]: Didn't you say we are using version 2 and not j?

File: src/main/java/frc/robot/commands/auton/SimpleAutonCommand.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands.auton;
+
+import edu.wpi.first.wpilibj2.command.ParallelDeadlineGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+import frc.robot.commands.drive.TankDriveCommand;
+import frc.robot.subsystems.DriveBase;
+
```
Comment (chit-sudeep) [inline src/main/java/frc/robot/commands/auton/SimpleAutonCommand.java:11]: There is no javaDoc here but there is one for arcadedrivecommand. Maybe you should add a javaDoc here.

**Comment by chit-sudeep** [comment]
Went over all the changes to the simpleAuton code and documentation.

Please review comments and make changes if necessary.

**Review by chit-sudeep — CHANGES_REQUESTED** [review]
Went over all the changes to the simpleAuton code and documentation.

Please review comments and make changes if necessary.

File: src/main/java/frc/robot/RobotContainer.java [hunk 3]
```diff
@@ -39,6 +40,6 @@ private void configureButtonBindings() {}
    * @return the command to run in autonomous
    */
   public Command getAutonomousCommand() {
-    return null;
+    return new SimpleAutonCommand(driveBase, Constants.SIMPLE_AUTON_SPEED, Constants.SIMPLE_AUTON_RUNTIME);
```
Comment (chit-sudeep) [inline src/main/java/frc/robot/RobotContainer.java:50]: Don't you need to import the constants at the top of the file to use them here?

File: src/main/java/frc/robot/RobotContainer.java (hunk 1 above)
Comment (aidan-mundy) [reply src/main/java/frc/robot/RobotContainer.java:8]: Notice on line 11 it says wpilibj2 for "command"? Only the "command" library has a V2

File: src/main/java/frc/robot/RobotContainer.java (hunk 3 above)
Comment (aidan-mundy) [reply src/main/java/frc/robot/RobotContainer.java:50]: RobotContainer.java is in the same package as Constants.java, so you do not need to import `Constants` in this file

**Review by chit-sudeep — DISMISSED** [review]
Looked over all changes, Aidan's comments, and new documentation.

Looks good.

**Review by kchenturtles — DISMISSED** [review]
Looked over added documentation and changes made by autoformatter
Looks good!

**Pull request merged**